metadata_annotation.ipynb

```diff
@@ -0,0 +1,2466 @@
+{
```

Comment:
<p>It's easier to review if you break up the markdown into a few more cells, but it's not a big deal.</p><p> </p><blockquote>This notebook demonstrates how to use pertpy to retrieve cell line metadata for perturbation annotation. </blockquote><p> </p><p>I think that it also does a bit more like developing and evaluating the model, right? I'd use this as the objective and the metadata fetching as the means to get to that objective.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>